### DISCUSSION TITLE
Question with the .TECH offer.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17704] [GitHub Education] [answered]
There is a student offer on the Github student developer pack for "get.tech". It promises One standard .TECH domain free for 1 year and 2 free email accounts with 100 MB free storage. Now, I'd assume that the email is permanent and the .TECH is only for 1 year. I don't know this for sure so help me!

Question: Are the 2 email given in .TECH student program permanent or only 1 year.

**Comment by mlarasusan** [comment]
Hello [ArchflexTech](https://github.com/ArchflexTech), you can contact partner support, techsquad@get.tech, for clarification about the offer. Thank you :]

**Comment by vincenttjia** [comment, marked as answer]
Hi @ArchflexTech I had claimed this offer in the past and to answer your question
No, the 2 emails are only for 1 year which will be unusable once the domain expire.
